lua/entities/ent_jack_turret_amateriel/shared.lua

```diff
@@ -2,6 +2,6 @@ ENT.Type = "anim"
 ENT.PrintName = "Anti-Materiel ST"
 ENT.Author = "Jackarunda"
 ENT.Category = "J.I. Defense Solutions"
-ENT.Spawnable = true
+ENT.Spawnable = false

```

Comment:
If we're getting rid of this for good, just delete it all. The materials, sounds, lua fules - all of it

Reply:
Applied, removed the ammo too.